packages/flutter_driver/lib/src/find.dart

```diff
@@ -26,7 +26,7 @@ abstract class CommandWithTarget extends Command {
   }
 
   /// Deserializes the command from JSON generated by [serialize].
-  CommandWithTarget.deserialize(Map<String, String> json)
+  CommandWithTarget.deserialize(Map<String, dynamic> json)
```

Comment:
In strong mode, json is generally Map<String, dynamic> not Map<String, String>
Outside of strong mode this code was allowed as you could cast
Map<String, dynamic> to Map<String, String> without getting an error.